instance/utils.py

```diff
@@ -133,3 +138,31 @@ def sufficient_time_passed(earlier_date, later_date, expected_days_since):
     """
     days_passed = (later_date - earlier_date).days
     return days_passed >= expected_days_since
+
+
+class DictTemplate(object):
+    """
+    Represents a dictionary based template for ansible configuration.
+    """
+
+    def __init__(self, path):
+        spec = importlib.util.spec_from_file_location(os.path.basename(path), path)
+        self.module = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(self.module)
```

Comment:
Why go through the trouble of importing manually based on a file name? Wouldn't it be simpler (and flexible) to pass the right function to get_template?